src/components/nav_drawer/nav_drawer.tsx

```diff
@@ -124,14 +191,14 @@ export class EuiNavDrawer extends Component {
 
     // Scrolls the menu and flyout back to top when the nav drawer collapses
     setTimeout(() => {
-      document.getElementById('navDrawerMenu').scrollTop = 0;
+      document.getElementById('navDrawerMenu')!.scrollTop = 0;
     }, 50);
 
     // In case it was locked before, remove the window resize listener
     window.removeEventListener('resize', this.functionToCallOnWindowResize);
   };
 
-  expandFlyout = (links, title, item) => {
+  expandFlyout = (links: FlyoutLinks[], title: string, item: EuiListGroupItemProps) => {
```

Comment:
These types I am unsure of.

Reply:
As this method is passed to EuiNavDrawerGroup's `flyoutMenuButtonClick`, it can be specified as

```suggestion
  expandFlyout: EuiNavDrawerGroupProps['flyoutMenuButtonClick'] = (links, title, item) => {
```